doc/source/installation/apt_repo.rst

```diff
@@ -12,13 +12,15 @@ Supported Releases:
 
  * 7.0 (wheezy)
  * 8.0 (jessie)
+ * 9.0 (stretch)
 
 * Ubuntu:
 
- * 12.04LTS (precise)
  * 14.04LTS (trusty)
  * 16.04LTS (xenial)
  * 16.10 (yakkety)
+ * 16.04 (zesty)
```

Comment:
17.04

Reply:
* 16.10 (yakkety) - shouldn't we remove it?